gtk4/src/icon_paintable.rs

```diff
@@ -0,0 +1,10 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use crate::IconPaintable;
+
+impl Default for IconPaintable {
```

Comment:
Why does this need a manual `Default` impl if it doesn't have a manual `new()`?

Reply:
You're right. Maybe IconPaintable should be on the list of #594 then? It's generally weird because the builder of IconPaintable has more options than its constructor. You can only construct it from a file but build it from a file and from an icon name...

Reply:
I guess let's keep IconPaintable as it is for now.